src/lib/message/L1ToL2Message.ts

```diff
@@ -66,6 +66,17 @@ export enum L1ToL2MessageStatus {
   EXPIRED = 5,
 }
 
+export enum EthDepositStatus {
+  /**
+   * Ethers are not deposited on L2 yet
+   */
+   NOT_YET_DEPOSITED_ON_L2 = 1,
```

Comment:
Maybe lets simplify these status names to "PENDING" and "DEPOSITED".

Also, should probably say "ETH" or "Eth" instead of "Ethers". I know how common it is to use ethers, but that's also the name of a js package, so probably best to avoid here.

Reply:
completely makes sense Chris! thanks!